i18n/ar.json

```diff
@@ -236,15 +237,15 @@
 	"smw_smwadmin_dbbutton": "استبداء أو ترقية الجداول",
 	"smw_smwadmin_announce": "أعلن عن هذه الويكي",
 	"smw_smwadmin_datarefresh": "إصلاح البيانات وتحديثها",
-	"smw_smwadmin_datarefreshdocu": "من الممكن استرجاع كل بيانات ميدياويكي الدلالية بناء على المحتويات الحالية للويكي.\nقد يكون هذا مفيدا لإصلاح البيانات المعطوبة أو لتحديث البيانات في حال تغيّرت الصيغة الداخلية بسبب ترقية برمجية.\nالتحديث يتم إجراؤه صفحة بصفحة ولن يتم لحظيا.\nالتالي يعرض ما إذا كان التحديث يجري و يسمح لك ببدء أو وقف التحديثات (إلا لو كانت هذه الخاصية قد عطّلها مدير الموقع).",
+	"smw_smwadmin_datarefreshdocu": "من الممكن استرجاع كل بيانات ميدياويكي الدلالية بناء على المحتويات الحالية للويكي.\nقد يكون هذا مفيدا لإصلاح البيانات المعطوبة أو لتحديث البيانات في حال تغيّرت الصيغة الداخلية بسبب ترقية برمجية.\nالتحديث يتم إجراؤه صفحة بصفحة ولن يتم لحظيا.\nالتالي يعرض ما إذا كان التحديث يجري و يسمح لك ببدء أو وقف التحديثات (إلا لو كانت هذه الوظيفة قد عطّلها مدير الموقع).",
 	"smw_smwadmin_datarefreshprogress": "<strong>يوجد تحديث جارٍ بالفعل.</strong>\nمن الطبيعي أن تتقدم عمليات التحديث ببطء بما أنها تجري على البيانات بكميات صغيرة كل مرة يستخدم فيها شخص ما الويكي.\nلإتمام التحديث سريعا يمكنك تشغيل سكربت صيانة ميدياويكي <code>runJobs.php</code> (استخدم الخيار <code>--maxjobs 1000</code> لتحديد عدد التحديثات المجراة في كل دفعة).\nالتقدم التقديري للتحديث الحالي:",
 	"smw_smwadmin_datarefreshbutton": "أبدا تحديث البيانات",
 	"smw_smwadmin_datarefreshstop": "أوقف هذا التحديث",
 	"smw_smwadmin_datarefreshstopconfirm": "نعم، أنا متأكد",
 	"smw_smwadmin_support": "الحصول على دعم",
 	"smw_smwadmin_supportdocu": "توجد مصادر متعددة للمساعدة في حال حدوث مشاكل:",
 	"smw_smwadmin_installfile": "لو واجهتك مشاكل مع تنصيبتك فابدأ بمراجعة الإرشادات في <a href=\"https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/docs/INSTALL.md#smw-installation\">ملف INSTALL</a>.",
-	"smw_smwadmin_smwhomepage": "وثائق المستخدم الكاملة لميدياويكي الدلالية موجودة في <b><a href=\"https://www.semantic-mediawiki.org\">semantic-mediawiki.org</a></b>.",
+	"smw_smwadmin_smwhomepage": "وثائق الاستخدام الكاملة لميدياويكي الدلالية موجودة في <b><a href=\"https://www.semantic-mediawiki.org\">www.semantic-mediawiki.org</a></b>.",
```

Comment:
Beauty lies always in the eyes of the beholder. However fixed for consistency. Thanks for noting.